RecoJets/JetProducers/plugins/FixedGridRhoProducerFastjetFromRecHit.cc

```diff
@@ -0,0 +1,164 @@
+/*
+
+Author: Swagata Mukherjee
+
+Date: November 2021
+
+This producer computes rho from ECAL and HCAL recHits. 
+The current plan is to use it in egamma and muon HLT, who currently use 
+the other producer called FixedGridRhoProducerFastjet.
+At HLT, FixedGridRhoProducerFastjet takes calotowers as input and computes rho.
+But calotowers are expected to be phased out sometime in Run3.
+So this recHit-based rho producer, FixedGridRhoProducerFastjetFromRecHit, can be used as an alternative.
+
+*/
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
+#include "DataFormats/Math/interface/LorentzVector.h"
```

Comment:
I think this is no longer needed. Can you check the other headers as well, if there was anything left over that's no longer used?